sprokit/src/sprokit/pipeline/process_factory.h

```diff
@@ -193,6 +194,49 @@ kwiver::vital::plugin_factory_vector_t const& get_process_list();
                                                  typeid( sprokit::process ).name(), \
                                                  sprokit::create_new_process< proc_type > ) )
 
+// ============================================================================
+class process_registrar
```

Comment:
Some basic doc strings for this class would be useful.

Reply:
done